### ISSUE TITLE
UCH Combat missions have become bugged

### ISSUE THREAD

**Issue by suntrip76** [RavienGaming/Reforged-Eden-2#343]
### Reforged Eden 2 Build Number

B22

### Game mode

Survival, Single Player

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

For some reason the UCH combat mission doesn't activate for me anymore. I have tried talking to the UCH captain who says I already have one, I then confirm to get it again. Nothing.
I have tried the Project Eden Mission Manager that does show that I can activate a combat mission to clear abandoned POIs but again, nothing when I try to activate.
I have tried the PDA debug in the Project Eden Admin page. Again, nothing. The console window show no warning or errors (or anything really) when trying to retake the mission.

I did a couple of the missions before the B22 update but since that one and a pda readdata as recommended in the patch notes, the combat missions are borked for me. That pda update also reset my missions to explore star systems.

I even tried doing a pda chapter reset and going way back to the first missions on Akua, doing those again, then the two main Project Eden mission lines, but this has not fixed the combat missions either (I thought it might have been a catch 22 where I had the combat missions unlocked but some prerequisite mission denied them from activating).

Can I reset the PDA completely perhaps?

**Comment by RavienGaming** [comment]
It's possible the pda readdata command got your missions stuck.
Try using the following console commands, in order, with about a 30 second wait in between each one.
The console log should mention the repeatable missions when you use the command (I did not test so hopefully I have the right chapter numbers).

pda brief 345
pda brief 346
pda clrchap 345
pda clrchap 346

Then completely restart Empyrion back to Windows desktop and see if you can take a new mission again. If it doesn't work the first time you may need to repeat the entire process again. Let me know if this does the trick or not please. Maybe I can add a different workaround if this doesn't work.
Note this may give you the mission rewards again, but you can just trash them if you want.

**Comment by suntrip76** [comment]
Did the commands. I had to deactivate the missions in between brief and clear as I "can't clear an active chapter".
Exited game completely and went in again. I'm near a captain that still tells me that I have an active mission and if I want another one. This particular captain tells me about legacy infected so I guess thats the mission she offers, but nothing happens when I accept it.
The pda brief command gives me the mission so I guess I can use that as a workaround for now.
I'm happy to do more testing as well, if you have other ideas :)

**Comment by suntrip76** [comment]
Also, I just tested accepting the mission, then deactivating it and going in the pda and try to activate it again with the Project Eden Mission Manager and nothing happens there either.

Added: Just to clarify, it does seem like the console commands work as intended:
![billede](https://github.com/user-attachments/assets/f97be774-4e92-4a45-91ee-e3f28a7e99da)

**Comment by suntrip76** [comment]
And this is what's in the Project Eden Mission Manager. It doesn't mention anything about bugs or the other zirax mission, just Clear POIs. 
![billede](https://github.com/user-attachments/assets/7168ab96-264a-4d77-9b3d-9414beb6f28f)

**Comment by RavienGaming** [comment]
Thanks I did find an error in the pda files. I will need to test if this was the actual issue or not though.

**Comment by suntrip76** [comment]
Update on this: After I completed one mission that I activated via the console, I tested the mission activator in the UCH mission manager and it said I didn't have an active mission now. And sure enough, the next captain I talked to gave me a mission just as it is intended. I also tried deactivating this new one and re-activating it in the mission manager and that works as well.

So I guess the solution if one becomes stuck like me is to console activate a mission, complete it and then it resets back to proper mode.
